core/src/parser/statement.rs

```diff
@@ -107,4 +107,55 @@ impl<'src> Parser<'src> {
 
         Ok(id)
     }
+
+    #[inline(always)]
+    pub fn break_statement(&mut self) -> Result<Node<'src>> {
+        let tok = peek!(self);
+        let statement = Item::BreakStatement {
+            label: match tok {
+                Semicolon | EndOfProgram => None,
+                _ => {
+                    if self.lexer.asi() {
+                        None
+                    } else {
+                        let label = expect_identifier!(self);
+                        let id = self.store(Item::Identifier(label.into()).at(0, 0));
+                        Some(id)
+                    }
+                }
+            }
+        };
+
```

Comment:
Thanks alot!
I use `expect_identifier` again, as it is also necessary in case of a labeled BreakStatement.